docs/step-by-step/kubeapps-on-tkg/README.md

```diff
@@ -0,0 +1,72 @@
+
+This tutorial will show you how to: i) successfully install Kubeapps with production-ready values, leveraging the existing identity management capabilities in TKG; ii) configure two application repositories: the public [VMware Marketplace™](https://marketplace.cloud.vmware.com/) and your private [VMware Tanzu™ Application Catalog™ (TAC)](https://tanzu.vmware.com/application-catalog) instance.
+
+## Assumptions and prerequisites
+
+As this guide will cover the installation of Kubeapps in a [VMware Tanzu™ Kubernetes Grid™](https://tanzu.vmware.com/kubernetes-grid) cluster and the configuration of an application repository for [VMware Tanzu™ Application Catalog™ (TAC)](https://tanzu.vmware.com/application-catalog), it assumes that you have access to a TKG cluster as well as a TAC pre-built or custom catalog. If you do not have access, please reach out to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html).
+
+This tutorial, therefore, assumes that:
+
+- You have a Tanzu™ Kubernetes Grid v1.3.1 or later cluster properly configured with identy management. Check out the [VMware Tanzu™ Kubernetes Grid 1.3 Documentation](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.3/vmware-tanzu-kubernetes-grid-13/GUID-index.html) and the [Enabling Identity Management in Tanzu Kubernetes Grid](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.3/vmware-tanzu-kubernetes-grid-13/GUID-mgmt-clusters-enabling-id-mgmt.html) guide.
+- You have access to the [VMWare Cloud Services Portal (CSP)](https://console.cloud.vmware.com/). If not, talk to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html) to request access.
+- You have access to, at least, the Tanzu™ Application Catalog™ Demo environment. If not, reach out to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html).
+- You have the _kubectl_ CLI and the Helm v3.x package manager installed. Learn how to [install kubectl and Helm v3.x](https://docs.bitnami.com/kubernetes/get-started-kubernetes/#step-3-install-kubectl-command-line).
+
+## Intended Audience
+
+This information is intended for administrators who want to install Kubeapps on a Tanzu Kubernetes Grid cluster and use it to deploy and manage applications from the VMware Marketplace™ and the VMware Tanzu™ Application Catalog™. Besides, this content is also intended for application administrators and developers who want to use Kubeaps to deploy and manage modern apps in a Kubernetes architecture.
+
+The information is written for users who have a basic understanding of Kubernetes and are familiar with container deployment concepts. In-depth knowledge of Kubernetes is not required.
+
+# Steps
+
+<!-- To be extracted to a separate file -->
+
+## Step 1: Prepare the identity management
```

Comment:
The first assumption states "TKG properly configured with identy management". If it's already configured, why do we need this step? Is it optional, or additional to the assumption already stated?

Reply:
Good gotcha here. It is something derived from this issue: https://github.com/kubeapps/kubeapps/issues/2764

We still have to figure out if we will use the same pinniped versions already included in TKG (currently is not an option since the version bundled is 0.4.1 - and we need >=0.6.0) or we will encourage users to upgrade the piniped version (if possible).

I'm gonna remove the misleading assumption anyway.